client/src/Components/ReviewsList.jsx

```diff
@@ -17,17 +18,20 @@ class ReviewsList extends React.Component {
     };
   }
 
-  componentDidMount() {
+  fetchReviews() {
     // Simple GET Request
     fetch(`/api/game/${this.props.gameId}/reviews/`)
       .then((res) => {
         if (!res.ok) {
           throw new Error("An error occurred while retrieving the review data. Please try again.");
         }
-
         return res.json(); 
       })
       .then((reviews) => {
+        // Ensure our Reviews are sorted chronologically
+        reviews.sort((reviewA, reviewB) => {
+          return new Date(reviewB.date_posted) - new Date(reviewA.date_posted);
+        });
         this.setState({ reviews });
       })
       .catch((err) => {
```

Comment:
Error is not handled yet.

Reply:
I've actually got a Tech Debt ticket for this here: https://trello.com/c/YUDHkF8Z

Hoping to circle back on that during the polish phase, for right now I'm considering it not Mission Critical. I can remove the comment if that's useful since it does seem like a loose end right now. What do you think?